packages/components/cascader-panel/src/index.vue

```diff
@@ -78,7 +78,7 @@ export default defineComponent({
     renderLabel: Function as PropType<RenderLabel>,
   },
 
-  emits: [UPDATE_MODEL_EVENT, CHANGE_EVENT, 'close', 'expand-change'],
+  emits: [UPDATE_MODEL_EVENT, CHANGE_EVENT, 'close', 'expandChange'],
```

Comment:
At present, emits mostly uses hyphens, and changing it to camelize may cause some problems

Reply:
It only affects the users who use `OnExpand-change` in JSX or render function, and should have no impact on SFC template and in-DOM template theoretically.  IMO, it doesn't break anything. Let me know if I was wrong.

Reply:
It's better to avoid such inconsistent behavior as much as possible